src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -19,18 +24,29 @@
     @Mock
     private ChipsRestClient chipsRestClient;
 
+    @Mock
+    private MessageProducer retryMessageProducer;
+
     @Mock
     private ApplicationLogger logger;
 
     @InjectMocks
     private MessageProcessorServiceImpl messageProcessorService;
 
     @Test
-    void processMessageTest() {
+    void processMessageTest() throws ServiceException {
```

Comment:
as it should be either/or for which route to take (chips or retry), I would also include a verify times(0) / never for the retryConsumer to verify it is not called.

Reply:
ok done